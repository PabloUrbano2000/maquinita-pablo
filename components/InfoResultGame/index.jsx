import React from 'react';
import Title from '../utils/Title';
import SubTitle from '../utils/SubTitle';
import TitleColor from '../utils/TitleColor';
import Box from '@mui/material/Box';
import Paper from '@mui/material/Paper';
import Button from '@mui/material/Button';

const InfoResultGame = () => {
  return (
    <Paper elevation={3} sx={{ padding: 5 }}>
      <Box
        sx={{
          display: 'flex',
          flexDirection: 'column',
          alignItems: 'center',
          gap: 5
        }}
        pl={'10%'}
        pr={'10%'}
      >
        <Box style={{ transform: 'rotate(-3deg)' }}>
          <TitleColor
            style={{
              fontSize: '2.3rem',
              display: 'block',
              marginBottom: 5,
              textAlign: 'center'
            }}
          >
            !FELICITACIONES!
          </TitleColor>
        </Box>
        <SubTitle
          style={{
            fontSize: '1.3rem',
            textAlign: 'center'
          }}
        >
          Eres el afortunado ganador en llevarse el{' '}
          <Title
            style={{
              fontSize: '1.5rem',
              textAlign: 'center'
            }}
          >
            PREMIAZO DE TROME.
          </Title>
        </SubTitle>

        <Title
          style={{
            display: 'block',
            fontSize: '1.5rem',
            textAlign: 'center',
            color: '#ffaf50'
          }}
        >
          NOMBRE Y APELLIDO
          <Title
            style={{
              display: 'block',
              fontSize: '1.5rem',
              textAlign: 'center',
              color: '#ffaf50'
            }}
          >
            DNI: 123123131
          </Title>
        </Title>

        <Title
          style={{
            display: 'block',
            fontSize: '1.3rem',
            textAlign: 'center'
          }}
        >
          Atento, porque estaremos enviando el código de validación a tu correo.
        </Title>
        <Button
          variant="contained"
          style={{
            fontSize: '1.3rem',
            borderRadius: 80
          }}
          onClick={() => console.log('envia')}
        >
          Enviar código al correo
        </Button>
      </Box>
    </Paper>
  );
};

export default InfoResultGame;
