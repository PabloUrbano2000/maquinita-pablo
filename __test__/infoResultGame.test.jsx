import { render } from '@testing-library/react';
import InfoResultGame from '../components/InfoResultGame';

it('render component InfoResultGame', () => {
  const { container } = render(<InfoResultGame />);
  expect(container).toMatchSnapshot();
});
