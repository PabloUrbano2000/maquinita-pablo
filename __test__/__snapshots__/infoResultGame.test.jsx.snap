// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component InfoResultGame 1`] = `
<div>
  <div
    class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation3 css-8ffmfy-MuiPaper-root"
  >
    <div
      class="MuiBox-root css-18z1sft"
    >
      <div
        class="MuiBox-root css-0"
        style="transform: rotate(-3deg);"
      >
        <span
          class="css-1sah1m0"
          style="font-size: 2.3rem; display: block; margin-bottom: 5px; text-align: center;"
        >
          !FELICITACIONES!
        </span>
      </div>
      <span
        class="css-8rchdx"
        style="font-size: 1.3rem; text-align: center;"
      >
        Eres el afortunado ganador en llevarse el
         
        <span
          class="css-yzbrn7"
          style="font-size: 1.5rem; text-align: center;"
        >
          PREMIAZO DE TROME.
        </span>
      </span>
      <span
        class="css-yzbrn7"
        style="display: block; font-size: 1.5rem; text-align: center; color: rgb(255, 175, 80);"
      >
        NOMBRE Y APELLIDO
        <span
          class="css-yzbrn7"
          style="display: block; font-size: 1.5rem; text-align: center; color: rgb(255, 175, 80);"
        >
          DNI: 123123131
        </span>
      </span>
      <span
        class="css-yzbrn7"
        style="display: block; font-size: 1.3rem; text-align: center;"
      >
        Atento, porque estaremos enviando el código de validación a tu correo.
      </span>
      <button
        class="MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButtonBase-root  css-sghohy-MuiButtonBase-root-MuiButton-root"
        style="font-size: 1.3rem; border-radius: 80px;"
        tabindex="0"
        type="button"
      >
        Enviar código al correo
        <span
          class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
        />
      </button>
    </div>
  </div>
</div>
`;
